Makefile.authx

```diff
@@ -75,6 +75,8 @@ init-authx: mkdir
 	$(MAKE) docker-volumes
 	echo "Setting up Keycloak"; \
 	source ${PWD}/lib/keycloak/keycloak_setup.sh; \
+	echo "Grabbing our local IP address"; \
+	source ${PWD}/setup_hosts.sh; \
```

Comment:
Why does this happen after the keycloak setup? Keycloak_setup calls curls and expects the /etc/hosts value to be set already.